foundations/installations/installations.md

```diff
@@ -196,10 +196,16 @@ After `sudo apt upgrade` runs for a while you will be asked whether you want to
 
 ### Step 4: Understand your new VM
 
+#### Step 4.1: Close TOP in the Windows browser and open the website in your VM
```

Comment:
Seems reasonable enough to me. Hopefully we'll put a dent in the amount of questions this concept generates on the Discord with this.